src/scribe_data/cli/main.py

```diff
@@ -224,7 +259,9 @@ def main() -> None:
         return
 
     if args.command in ["list", "l"]:
-        list_wrapper(args.language, args.data_type, args.all)
+        list_wrapper(
+            language=args.language, data_type=args.data_type, all_bool=args.all
```

Comment:
Thanks for adding in the args! Really appreciate explicit function calls to make the functionality more clear.